instance/tests/integration/integration_instance.py

```diff
@@ -263,6 +247,24 @@ def test_activity_csv(self):
         # stdout should contain 3 lines (as opposed to 2) to account for the last newline.
         self.assertEqual(len(out_lines), 3)
 
+    ''' FIXME JV : Need to add an additional test container?
```

Comment:
I tried using `shard 0` here instead, but it disrupted the other integration tests.

So 68bb9a8 comments out this test - @smarnach Do I have to need another container to the CircleCI project?